core/core-compilation.el

```diff
@@ -0,0 +1,35 @@
+;;; core-compilation.el --- Spacemacs Core File
+;;
+;; Copyright (c) 2012-2021 Sylvain Benner & Contributors
+;;
+;; Author: Eugene "JAremko" Yaremenko <w3techplayground@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst spacemacs-compiled-files
```

Comment:
Should also mention that `packed.el` and `auto-compile.el` are omitted on purpose.   They are loaded before we can check if they are up to date.